test/e2e/network/network_policy.go

```diff
@@ -1744,3 +1747,69 @@ func cleanupNetworkPolicy(f *framework.Framework, policy *networkingv1.NetworkPo
 		framework.Failf("unable to cleanup policy %v: %v", policy.Name, err)
 	}
 }
+
+var _ = SIGDescribe("SCTP NetworkPolicy [Feature:SCTP] [LinuxOnly]", func() {
```

Comment:
it's kind of random to put the `[Feature:SCTP]` here but the `[Feature:NetworkPolicy]` in the test case...

Reply:
Fixed.